### PR TITLE
feat(Digital Shadows): Add playbook template

### PR DESCRIPTION
**Pull request by PierrickV**
Screenshot from 2022-03-18 12-08-40

### CODE REVIEW

**Comment by PierrickV** [comment]
Associated issue https://github.com/SekoiaLab/platform/issues/24603

File: playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        "type": "trigger",
+        "outputs": {
+          "default": [
+            "1"
+          ]
+        },
+        "module_uuid": "e76687ed-db66-482a-8549-f3ef3b248e06",
+        "trigger_uuid": "7d026824-2c7c-4477-bc36-a1f96837eba1",
+        "module_configuration_uuid": "5dde9166-4727-4125-b879-6a1c5465aca2",
+        "trigger_configuration_uuid": "f8bac705-3ce1-447c-b9fe-4063653f73a5"
```
Comment (LeoArurault) [inline playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json:17]: Configurations depends on your community
```suggestion
```

File: playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "outputs": {
+          "default": []
+        },
+        "arguments": {
+          "event": "{{ node.0['alerts'] }}",
+          "intake_server": "https://intake.sekoia.io"
+        },
+        "action_uuid": "10314466-ac0c-4067-8e81-4093e3d1e1da",
+        "module_uuid": "92d8bb47-7c51-445d-81de-ae04edbb6f0a",
+        "module_configuration_uuid": "4e9bfc9f-a2e4-4bc1-9f6d-cb81b6281d57"
```
Comment (LeoArurault) [inline playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json:32]: Same here
```suggestion
```

File: playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      }
+    },
+    "workspace": "Operation Center",
+    "description": "Synchronise alerts from Digital Shadows SearchLight to SEKOIA.IO events",
+    "community_uuid": "02ff3284-506e-49d6-a9f5-99dbb2ea69ed"
```
Comment (LeoArurault) [inline playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json:37]:
```suggestion
```

**Review by LeoArurault — CHANGES_REQUESTED** [review]

File: playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Fetch alerts from Digital Shadows SearchLight",
+    "uuid": "367ead9d-9e0f-42e7-abc2-1c134bb95a3e",
```
Comment (LeoArurault) [inline playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json:3]: I forgot that one on my first review
```suggestion
```

**Review by LeoArurault — CHANGES_REQUESTED** [review]
Maybe you could add some tags. @squioc used `events` one for example

File: playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json (hunk 4 above)
Comment (PierrickV) [reply playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json]: Review applied in commit 475c02a01f26e5584f937eba5ce3ec6710350d0d

**Review by LeoArurault — APPROVED** [review]

**Pull request merged**